### ISSUE TITLE
Wrong mysqli_error argument when not able to connect

### ISSUE THREAD

**Issue by bpolaszek** [dshafik/php7-mysql-shim#50]
```
Warning: mysqli_error() expects parameter 1 to be mysqli, bool given in ./vendor/dshafik/php7-mysql-shim/lib/mysql.php on line 235

Could not connect to database
```

Thank you,
Ben

**Comment by dshafik** [comment]
This error is probably improved by #53, can you check and/or provide more info?

**Comment by dshafik** [comment]
Assumed closed by #53
